unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -154,8 +154,19 @@ public static void main(String[] args) throws Exception {
                             "Could not find or create " + options[OUTPUTFOLDER].value);
                 }
             } else {
-                throw new IllegalArgumentException(
-                        "Missing command line option: --outputfolder (or -f)");
+                try {
+                    destinationFolder = new File(Settings.Output.GEN_DIR + "uax42\\fragments\\");
```

Comment:
let's please use forward slashes, so that this works on non-Windows machines, or use Java APIs to build the path from pieces; Settings.java should have some useful code to look at

Reply:
Fixed. (changing to forward slashes was sufficient)